implementations/wrapscan-registry/polywrap.yaml

```diff
@@ -0,0 +1,9 @@
+format: 0.4.0
+project:
+  name: wrapscan-registry-resolver-ext
```

Comment:
Maybe we should name this wrapscan-resolver-ext (and the dir wrapscan, or wrapscan-gateway)
I'm thinking that because this resolver can be used for any URI the gateway can resolve (e.g. ens, ipfs, in the future) not just registry URIs.
It might be premature optimization, but if we wanted to switch later, it would involve renaming this meaning the URI of deployment changes and so we would need to re-release the clients